tldw_Server_API/tests/AuthNZ/integration/test_evaluations_permissions_claims.py

```
@@ -0,0 +1,185 @@
+            org_ids=[],
+            team_ids=[],
+        )
+    )
+
+    with TestClient(app) as client:
+        resp = client.get("/api/v1/evaluations/")
+    assert resp.status_code in (401, 403)
+
+
+@pytest.mark.asyncio
+async def test_evaluations_list_forbidden_without_claims(monkeypatch):
```

Ruff (0.14.6)

Unused function argument: `monkeypatch`

(ARG001)